addOns/fuzz/src/main/javahelp/org/zaproxy/zap/extension/fuzz/resources/help/contents/api.html

```diff
@@ -0,0 +1,173 @@
+        <br class="SCXW166475849 BCX2">
+      </span><span class="TextRun SCXW166475849 BCX2" data-contrast="none" style="color: rgb(0, 0, 0); font-size: 10pt; font-family: Consolas, Consolas_MSFontService, monospace; line-height: 16px;" lang="EN-AU"><span class="NormalTextRun SCXW166475849 BCX2" style="background-color: inherit;">&nbsp; &nbsp; &nbsp; ]</span></span><span class="LineBreakBlob BlobObject DragDrop SCXW166475849 BCX2" style="font-size: 10pt; line-height: 16px; font-family: Consolas, Consolas_MSFontService, monospace;"><span class="SCXW166475849 BCX2">&nbsp;</span>
+        <br class="SCXW166475849 BCX2">
+      </span><span class="TextRun SCXW166475849 BCX2" data-contrast="none" style="color: rgb(0, 0, 0); font-size: 10pt; font-family: Consolas, Consolas_MSFontService, monospace; line-height: 16px;" lang="EN-AU"><span class="NormalTextRun SCXW166475849 BCX2" style="background-color: inherit;">&nbsp; &nbsp; }</span></span><span class="LineBreakBlob BlobObject DragDrop SCXW166475849 BCX2" style="font-size: 10pt; line-height: 16px; font-family: Consolas, Consolas_MSFontService, monospace;"><span class="SCXW166475849 BCX2">&nbsp;</span>
+        <br class="SCXW166475849 BCX2">
+      </span><span class="TextRun SCXW166475849 BCX2" data-contrast="none" style="color: rgb(0, 0, 0); font-size: 10pt; font-family: Consolas, Consolas_MSFontService, monospace; line-height: 16px;" lang="EN-AU"><span class="NormalTextRun SCXW166475849 BCX2" style="background-color: inherit;">&nbsp; ]</span></span><span class="LineBreakBlob BlobObject DragDrop SCXW166475849 BCX2" style="font-size: 10pt; line-height: 16px; font-family: Consolas, Consolas_MSFontService, monospace;"><span class="SCXW166475849 BCX2">&nbsp;</span>
+        <br class="SCXW166475849 BCX2">
+      </span><span class="TextRun SCXW166475849 BCX2" data-contrast="none" style="color: rgb(0, 0, 0); font-size: 10pt; font-family: Consolas, Consolas_MSFontService, monospace; line-height: 16px;" lang="EN-AU"><span class="NormalTextRun SCXW166475849 BCX2" style="background-color: inherit;">}</span></span><span class="EOP SCXW166475849 BCX2" data-ccp-props="{&quot;201341983&quot;:0,&quot;335559739&quot;:0,&quot;335559740&quot;:240,&quot;469777462&quot;:[916,1832,2748,3664,4580,5496,6412,7328,8244,9160,10076,10992,11908,12824,13740,14656],&quot;469777927&quot;:[0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0],&quot;469777928&quot;:[1,1,1,1,1,1,1,1,1,1,1,1,1,1,1,1]}" style="font-size: 10pt; line-height: 16px; font-family: Consolas, Consolas_MSFontService, monospace;">&nbsp;</span></p>
+    </div>
+</div>
+
+<p>This way the fuzzer accepts multiple fuzz locations and multiple payload types.</p>
+<p>More of these examplesand the JSON schema can be found at zap-api-docs:&nbsp;</p>
+<p><a href="https://github.com/zaproxy/zap-api-docs/fuzz">https://github.com/zaproxy/zap-api-docs/fuzz</a>/</p>
+<p>The JSON schema is version controlled and the json should be updated according to the version of the fuzzer being used.</p>
```

Comment:
If this is true should the parsing to moved to a utility class with appropriate methods (instead of being included monolithically in FuzzAPI).

Reply:
Just removed the version controlling, not necessary, works well.